### PR TITLE
Add blockchain addresses to the screener

### PR DESCRIPTION
**Pull request by IvanIvanoff**
## Changes
- Add `Balance.current_balance_top_addresses/4`
- Improve the speed of ETH/ERC20 current_balance by using the realtime balances table
- Add blockchain addresses to the screener by balance and labels
- Add `dynamicWatchlist` API (see example). It combines the currently used `allProjectsByFunction` functionality where you can fetch projects by function without a watchlist and also allow to do this with blockchain addresses. See examples below.
- Add `address_selector` that works similar to the projects' `selector`. See examples below.

Example:
```graphql
mutation {
      createWatchlist(
        type: BLOCKCHAIN_ADDRESS
        name: "My list"
        color: BLACK
        function: "{\"args\":{\"filters\":[{\"args\":{\"labels\":[\"centralized_exchange\",\"decentralized_exchange\",\"CEX Trader\"],\"page\":1,\"page_size\":20,\"slug\":\"ethereum\"},\"name\":\"top_addresses\"}]},\"name\":\"address_selector\"}"
        isScreener: false
        ) {
         id
         name
         type
         color
         isPublic
         isScreener
         user{ id }
         listItems{
           blockchainAddress { address infrastructure labels { name origin }  }
    }
  }
}
```

Example:
```graphql
mutation {
  createWatchlist(
    type: BLOCKCHAIN_ADDRESS
    name: "My list"
    color: BLACK
    function: "{\"args\":{\"filters\":[{\"args\":{\"page\":1,\"page_size\":10,\"slug\":\"ethereum\"},\"name\":\"top_addresses\"},{\"args\":{\"page\":1,\"page_size\":10,\"slug\":\"santiment\"},\"name\":\"top_addresses\"}],\"filters_combinator\":\"or\"},\"name\":\"address_selector\"}"
    isScreener: false
    ) {
      id
      name
      type
      color
      isPublic
      isScreener
      user{ id }
      listItems{
        blockchainAddress { address infrastructure labels { name origin } }
      }
  }
}
```

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: lib/sanbase_web/graphql/resolvers/user/user_list_resolver.ex [hunk 1]
```diff
@@ -43,6 +43,16 @@ defmodule SanbaseWeb.Graphql.Resolvers.UserListResolver do
     UserList.fetch_all_public_lists(type)
   end
 
+  # def dynamic_watchlist(_root, %{} = args, _resolution) do
```
Comment (tspenov) [inline lib/sanbase_web/graphql/resolvers/user/user_list_resolver.ex:46]: commented code left

File: lib/sanbase_web/graphql/schema/queries/user_list_queries.ex [hunk 2]
```diff
@@ -78,11 +78,19 @@ defmodule SanbaseWeb.Graphql.Schema.UserListQueries do
       resolve(&UserListResolver.user_list/3)
     end
 
+    # field :dynamic_watchlist, :user_list do
```
Comment (tspenov) [inline lib/sanbase_web/graphql/schema/queries/user_list_queries.ex:81]: why is this commented ?

Comment (IvanIvanoff) [reply lib/sanbase_web/graphql/schema/queries/user_list_queries.ex:81]: I need first check if we need this and then implement it in another PR

Comment (IvanIvanoff) [reply lib/sanbase_web/graphql/schema/queries/user_list_queries.ex:81]: @tspenov I think we can leave it like this so we don't forget to do it. And also to not delete the types so we don't have to write them again.

**Review by tspenov — APPROVED** [review]

**Pull request merged**